readme.txt

```diff
@@ -1,8 +1,8 @@
-=== Split Domain ===
+=== Switchboard ===
 Contributors: mboynes, alleyinteractive
 Tags: domains, multiple domains, multiple sites
```

Comment:
I wonder if `syndication` might be a relevant search tag for this plugin, if you ever put it in the directory. When I was at WC Publishers the idea of creating a network of sites and selectively syndicating content out to them was discussed numerous times. This seems like it would scratch that itch for people who don't need the complexity of a multisite network.

Reply:
Good idea, thanks!